plugins/UsersManager/API.php

```diff
@@ -879,6 +879,11 @@ private function enrichUser($user)
         unset($user['password']);
         unset($user['ts_password_modified']);
 
+        if($lastSeen = LastSeenTimeLogger::getLastSeenTimeForUser($user['login']))
+        {
```

Comment:
minor coding style improvement, so it matches the other code
```suggestion
        if ($lastSeen = LastSeenTimeLogger::getLastSeenTimeForUser($user['login'])) {
```